src/modules/content-tree/content.tree.module.js

```diff
@@ -2,20 +2,79 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import List from './components/list/list.component';
 
+const CLASS_IS_TREE_RESIZING = 'ez-is-tree-resizing';
+
 export default class ContentTreeModule extends Component {
     constructor(props) {
         super(props);
 
+        this.changeContainerWidth = this.changeContainerWidth.bind(this);
+        this.addWidthChangeListener = this.addWidthChangeListener.bind(this);
+        this.removeWidthChangeListener = this.removeWidthChangeListener.bind(this);
+        this._refTreeContainer = React.createRef();
+
         this.state = {
             selectedLocationId: props.selectedLocationId,
             locations: props.preloadedLocations,
+            resizeStartPositionX: 0,
+            containerWidth: 0,
+            resizedContainerWidth: 0,
+            isResizing: false,
         };
     }
 
+    componentWillUnmount() {
+        this.clearDocumentResizingState();
```

Comment:
It doesn't have much impact, because we just want to make sure the listeners and a CSS class is removed while unmounting a module. If there's no listener set then browser won't do any additional action despite of invoking this function once.